packages/zowe-explorer/src/configuration/Profiles.ts

```diff
@@ -126,10 +128,7 @@ export class Profiles extends ProfilesCache {
                 await AuthUtils.errorHandling(error, { profile: theProfile });
                 return profileStatus;
             }
-        } else if (
-            !usingTokenAuth &&
-            !((theProfile.profile.user && theProfile.profile.password) || (theProfile.profile.certFile && theProfile.profile.certKeyFile))
-        ) {
+        } else if (!usingTokenAuth && !(usingBasicAuth || usingCertAuth)) {
```

Comment:
Can we expand the `!(A OR B)` into `!A AND !B`?
That will make it a bit more consistent with the check below 🙏